tools/pi-appliance-update

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+UPDATE_TYPE="security"
+BOOT=0
+CLEAN="false"
+HELP=0
+
+while getopts "t:bch" option; do 
+ case "${option}" 
+ in 
+ t) UPDATE_TYPE=${OPTARG};; 
+ b) BOOT=1;; 
+ c) CLEAN="true";;
+ h) HELP=1;;
+ esac 
+done 
+
+if [ $HELP -eq 1 ]; then
+   echo "Usage: $0 [-b] [-c] [-t type]"
+   echo ""
+   echo " -b        : boot after update if necessary."
+   echo " -c        : clean unused packages after update."
+   echo " -t <type> : Either update 'security' packages or 'updates'."
+   exit 0
+fi
+
+LOGFILE=/var/log/privacyidea/update-${UPDATE_TYPE}.log
+
+function log {
+	echo $1 >> $LOGFILE
```

Comment:
Hmm, what about also echoing this to stdout?
Right now, the script is extremely quiet :-)
```
(venv) root@appliance1:/mnt/privacyidea-appliance# ./tools/pi-appliance-update 
Extract templates from packages: 100%
(venv) root@appliance1:/mnt/privacyidea-appliance#
```

Reply:
Usually this script will run in a cronjob. So why should we echo this to stdout?

Reply:
Ah OK, I assumed this was also meant to be run manually.